connect_ext_ppr/events.py

```diff
@@ -42,7 +43,7 @@ def handle_listing_processing(self, request):
         if request['status'] == 'listed':
             mp = get_marketplaces(self.client, [request['contract']['marketplace']['id']]).first()
             request['contract']['marketplace'] = mp
```

Comment:
There is a possibility to reuse the add_deployments function here. Here we are doing it for a single listing and in add_deployments we are doing the same action for a list of listings.

Reply:
Agree, lets do it!